index.html

```diff
@@ -30,6 +30,18 @@
 
 		</style>
 		<script type="text/javascript">
+			window.onload = function(){
```

Comment:
nit pick, could this have a space character between the `)` and the `{`?

Reply:
Done